modules/core/shared/src/test/scala/gem/math/WavelengthSpec.scala

```diff
@@ -38,8 +46,62 @@ final class WavelengthSpec extends CatsSuite {
 
   test("Construction from an arbitrary Int must not allow negative values") {
     forAll { (n: Int) =>
-      Wavelength.fromAngstroms.getOption(n).isDefined shouldEqual n >= 0
+      Wavelength.fromPicometers.getOption(n).isDefined shouldEqual n >= 0
+    }
+  }
+
+  private def pow10(exp: Int): Int =
+    BigInt(10).pow(exp).toInt
```

Comment:
Why `BigInt`?

Reply:
Surprisingly there is no integer version of `Math.pow` and I wanted to avoid using the `Double` version and rounding, etc.  I guess it could do though.